.github/workflows/generate-docs.yml

```diff
@@ -18,22 +18,8 @@ jobs:
       - name: Check out the repo
         uses: actions/checkout@v2.4.0
 
-      - name: Install Poetry
-        uses: snok/install-poetry@v1.3
-        with:
-          virtualenvs-create: true
-          virtualenvs-in-project: true
-
-      - name: Load cached venv
-        id: cached-poetry-dependencies
-        uses: actions/cache@v2
-        with:
-          path: .venv
-          key: venv-${{ runner.os }}-${{ matrix.python-version }}-${{ hashFiles('**/poetry.lock') }}
-
       - name: Install dependencies
-        run: poetry install
-        if: steps.cached-poetry-dependencies.outputs.cache-hit != 'true'
+        run: pip install requirements.txt
 
       - name: Generate documentation
         run: bash ./build_mkdocs.sh
```

Comment:
As far as I can see there is no build_mkdocs.sh in the repository ! (neither any mkdocs documentation, ain't it ?)